packages/common/src/helpers.ts

```diff
@@ -32,3 +33,20 @@ export function useFilesToBeExempted(): string[] {
     }
     return [];
 }
+
+export function useCustomRules(): string[] {
+    const filePath = process.env.SA11Y_CUSTOM_RULES ?? '';
+    if (filePath !== '') {
+        try {
+            // Read the file synchronously
+            const data = fs.readFileSync(filePath, 'utf-8');
```

Comment:
please use async file read

Reply:
I tried that but It gives a warning "(!) Missing shims for Node.js built-ins
Creating a browser bundle that depends on "fs/promises".